README.md

```diff
@@ -151,8 +153,14 @@ To run the test container use the following commands in the terminal:
 Build:
 docker build --target test --tag todo_app:test .  
 
-Run:
-docker run --env-file .env.test todo_app:test 
+Run integration tests:
+docker run --env-file .env.test todo_app:test todo_app/tests/test_app.py
+
+Run unit tests:
+docker run --env-file .env.test todo_app:test todo_app/tests/test_unit.py
+
+Run E2E tests:
+docker run -e TRELLO_API_KEY=${{secrets.TRELLO_API_KEY}} -e TRELLO_API_TOKEN=${{secrets.TRELLO_API_TOKEN}} -e SECRET_KEY=anything todo_app:test todo_app/tests/test_e2e.py
```

Comment:
This won't work locally! There are no `secrets` set on the local machine, instead we might want to just pass through the `.env` file that we've told users to set up, e.g.
```
docker run --env-file .env todo_app:test todo_app/tests/test_e2e.py
```